addons/main/XEH_postInit.sqf

```diff
@@ -77,19 +77,22 @@ if (GVAR(aceMedicalLoaded)) then {
             _unit setVariable [QGVAR(HandleDamageEHID), _id];
         };
 
+        if (_unit getVariable [QGVAR(HandleHealEHID), -1] isEqualTo -1) then {
+            private _id = _unit addEventHandler ["HandleHeal", {
+                [{
+                    params ["_unit", "_healer"];
+                    _unit setDamage 0;
+                    [_unit, _healer, true] call FUNC(handleHealEh);
+                }, _this, 5] call CBA_fnc_waitAndExecute;
+                true
+            }];
+            _unit setVariable [QGVAR(HandleHealEHID), _id];
+        };
+
```

Comment:
huh why tho? the post init should only run once per machine. This is not a displayed EH which can persists between previews or sessions.

Reply:
I copied what you did above this for HandleDamage, which might not need it either I guess.